common/tracing/src/logging.rs

```diff
@@ -113,6 +113,29 @@ pub fn init_global_tracing(app_name: &str, dir: &str, level: &str) -> Vec<Worker
     guards
 }
 
+pub fn init_query_logger(
+    log_name: &str,
+    dir: &str,
+) -> (Vec<WorkerGuard>, Arc<dyn Subscriber + Send + Sync>) {
+    let mut guards = vec![];
+
+    let rolling_appender = RollingFileAppender::new(Rotation::HOURLY, dir, log_name);
```

Comment:
I think meta service also needs the rolling appender, maybe add it in further PR